### DISCUSSION TITLE
Backport aarch64-related issues to 1.10

### DISCUSSION THREAD

**Discussion by dshevtsov-c** [tarantool/tarantool#8085] [Q&A] [answered]
Hello, there are several issues related to aarch64 that were resolved for 2.x release in the scope of https://github.com/tarantool/tarantool/issues/5629. Do you have any plans to backport them to 1.10.x?

**Comment by igormunkin** [comment, marked as answer]
There are no such plans for now. ARM64 support has been started with version [2.10](https://github.com/tarantool/tarantool/releases/tag/2.10.0-beta1).

**Reply by dshevtsov-c** [reply]
Can you think of any issues related to aarch64 other than those listed at https://github.com/tarantool/tarantool/issues/5629?
